### DISCUSSION TITLE
Problem configuring for my callsign and lat/lon after update - stuck on a default version of sim.cfg

### DISCUSSION THREAD

**Discussion by W7DGP** [alanbjohnston/CubeSatSim#304] [General]
current sim.cfg:
W7DGP 19  47.7466 -122.1940 no 3 434.9000 435.0000 no

Before updating, it sent my call sign.  After updating it sends AMSAT. It does not read this sim.cfg file. When I use config -c, it says W7DGP is in there so I typed it in again. No change after reboot. Later, I tried a total power off reboot and no change. Still AMSAT. 
Did the update create another sim.cfg? I don't see one. Did the update fail and I didn't notice it? How can you tell which version you are running? 
config is working - I can change mode from BPSK to FSK for example and check the I2C, etc. 
Checking the log file, I see this:
 Config file /home/pi/CubeSatSim/sim.cfg contains AMSAT 1 0.000000 0.000000 no 3 434.9000 435.0000 no
Note that its still AMSAT and my GPS coordinates are not changed either. The default looks like on the equator at the Greenwich meridian!
How can I get it to read the current sim.cfg file at boot up?
Thanks, Larry W7DGP

**Comment by alanbjohnston** [comment]
Hi Larry,

Something definitely went wrong in your update!  If you could log into your Pi and type in these commands and copy and paste the output here, that would help me figure out what went wrong.  The command `ls` is the letter L and the letter S:

cd
pwd
ls -all
ls -all CubeSatSim
cd CubeSatSim
cat sim.cfg
git status
./config
./log

Thanks,
Alan 
KU2Y

**Reply by W7DGP** [reply]
Thanks Alan, here it is:

pi@cubesatsimbeta:~ $ cd
pi@cubesatsimbeta:~ $ pwd
/home/pi
pi@cubesatsimbeta:~ $ ls -all
total 60
drwxr-xr-x  8 pi   pi    4096 Jan 27 07:45 .
drwxr-xr-x  3 root root  4096 Jan 27  2022 ..
-rw-------  1 pi   pi     721 Feb 29 15:12 .bash_history
-rw-r--r--  1 pi   pi     220 Jan 27  2022 .bash_logout
-rw-r--r--  1 pi   pi    3523 Jan 27  2022 .bashrc
drwx------  4 pi   pi    4096 Jan 27 07:49 .config
-rw-r--r--  1 pi   pi      23 Mar  5  2022 .gitconfig
-rw-r--r--  1 pi   pi     807 Jan 27  2022 .profile
drwxr-xr-x 13 pi   pi    4096 Feb 28 13:16 CubeSatSim
drwxr-xr-x  3 pi   pi    4096 Feb 20  2022 PiSSTVpp
drwxr-xr-x  9 pi   pi   12288 Feb 20  2022 direwolf
drwxr-xr-x  4 pi   pi    4096 Feb 28 13:15 pi-power-button
drwxr-xr-x  7 pi   pi    4096 Feb 20  2022 rpitx
pi@cubesatsimbeta:~ $ ls -all CubeSatSim
total 2372
drwxr-xr-x 13 pi   pi     4096 Feb 28 13:16 .
drwxr-xr-x  8 pi   pi     4096 Jan 27 07:45 ..
drwxr-xr-x  8 pi   pi     4096 Feb 28 13:15 .git
-rw-r--r--  1 pi   pi      296 Feb 20  2022 .gitignore
-rw-r--r--  1 pi   pi        2 Feb 29 14:04 .mode
-rw-r--r--  1 pi   pi      736 Feb 28 13:15 .updated
-rw-r--r--  1 pi   pi    10993 Feb 20  2022 Makefile
-rw-r--r--  1 pi   pi     4779 Feb 28 13:15 README.md
-rw-r--r--  1 pi   pi     6424 Feb 20  2022 TelemEncoding.c
-rw-r--r--  1 pi   pi    11446 Feb 20  2022 TelemEncoding.h
-rw-r--r--  1 pi   pi     4680 Feb 20  2022 TelemEncoding.o
drwxr-xr-x  2 pi   pi     4096 Feb 20  2022 afsk
drwxr-xr-x 10 pi   pi     4096 Feb 20  2022 ax5043
-rw-r--r--  1 pi   pi        1 Dec 19 06:31 command_count.txt
-rw-r--r--  1 pi   pi        7 Dec 19 06:13 command_tx
-rwxr-xr-x  1 pi   pi    15625 Feb 28 13:15 config
-rwxr-xr-x  1 pi   pi    87456 Feb 28 13:15 cubesatsim
-rw-r--r--  1 pi   pi    35149 Feb 20  2022 gpl.txt
drwxr-xr-x  8 pi   pi     4096 Feb 20  2022 groundstation
drwxr-xr-x 11 pi   pi     4096 Dec 19 06:13 hardware
-rw-r--r--  1 root root     15 Feb 29 15:12 id.txt
-rw-r--r--  1 pi   pi    13475 Dec 19 06:13 ina219.py
-rwxr-xr-x  1 pi   pi     6995 Dec 19 06:13 install
lrwxrwxrwx  1 pi   pi        7 Feb 20  2022 install.sh -> install
-rw-r--r--  1 pi   pi    47128 Feb 20  2022 libax5043.a
-rwxr-xr-x  1 pi   pi      420 Feb 20  2022 log
-rw-r--r--  1 pi   pi   887171 Feb 21 14:42 log.txt
-rw-r--r--  1 pi   pi    84501 Feb 28 13:15 main.c
-rw-r--r--  1 pi   pi     5075 Feb 28 13:15 main.h
-rw-r--r--  1 pi   pi    65800 Feb 28 13:15 main.o
-rw-r--r--  1 root root 589484 Feb 29 15:12 morse.wav
-rw-r--r--  1 pi   pi    24541 Feb 28 13:15 rpitx.py
-rw-r--r--  1 pi   pi       53 Feb 29 15:12 sim.cfg
drwxr-xr-x  7 pi   pi     4096 Feb 20  2022 spacecraft
drwxr-xr-x  2 pi   pi     4096 Feb 20  2022 spreadsheet
drwxr-xr-x  2 pi   pi     4096 Feb 20  2022 sstv
-rw-r--r--  1 pi   pi    33810 Feb 20  2022 sstv_image_1_320_x_256.jpg
-rw-r--r--  1 pi   pi    22803 Feb 20  2022 sstv_image_2_320_x_256.jpg
drwxr-xr-x 10 pi   pi     4096 Dec 19 06:13 stempayload
drwxr-xr-x  2 pi   pi     4096 Feb 20  2022 systemd
-rw-r--r--  1 root root    165 Feb 21 14:35 t.txt
-rwxr-xr-x  1 pi   pi    16548 Dec 19 06:13 telem
-rw-r--r--  1 pi   pi     4475 Dec 19 06:13 telem.c
-rw-r--r--  1 pi   pi     8636 Dec 19 06:13 telem.o
-rw-r--r--  1 root root 167606 Mar  1 13:52 telem.txt
-rw-r--r--  1 root root 115816 Feb 21 14:35 telem.wav
-rw-r--r--  1 root root     18 Mar  1 13:52 telem_string.txt
-rwxr-xr-x  1 pi   pi     7570 Dec 19 06:13 update
lrwxrwxrwx  1 pi   pi        6 Feb 20  2022 update.sh -> update
drwxr-xr-x  2 pi   pi     4096 Feb 20  2022 wav
pi@cubesatsimbeta:~ $ cat sim.cfg
cat: sim.cfg: No such file or directory
pi@cubesatsimbeta:~ $ cat ./sim.cfg
cat: ./sim.cfg: No such file or directory
pi@cubesatsimbeta:~ $ git status
fatal: not a git repository (or any of the parent directories): .git
pi@cubesatsimbeta:~ $ ./config
-bash: ./config: No such file or directory
pi@cubesatsimbeta:~ $ ./log
-bash: ./log: No such file or directory
pi@cubesatsimbeta:~ $

**Comment by alanbjohnston** [comment]
Thanks for doing that!  I don't see anything wrong so far in the commands that worked.

The "No such file or directory" errors were because you missed a command.  Type this and it should give the results:

cd
cd CubeSatSim
cat sim.cfg
git status
./config
./log

Thanks,
Alan
KU2Y

**Comment by W7DGP** [comment]
I see that now.  I’ve always been able to cat that file before. It normally shows updated info like my call and lat/lon.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by alanbjohnston** [comment]
Larry, please type these commands and paste the output here so I can continue debugging:

cd
cd CubeSatSim
cat sim.cfg
git status
./config
./log

Thanks,
Alan
KU2Y

**Reply by W7DGP** [reply]
Ok, here it is. I had to do it in pieces. When I execute ./log its such a long listing I can't go back and copy/paste it. I could pipe it into a file and send it that way I think. Anyway, here is the first part up to and including the ./config command:
pi@cubesatsimbeta:/home $ cd
pi@cubesatsimbeta:~ $ cd CureSatSim
-bash: cd: CureSatSim: No such file or directory
pi@cubesatsimbeta:~ $ cd CubeSatSim
pi@cubesatsimbeta:~/CubeSatSim $ cat sim.cfg
W7DGP 21  47.7466 -122.1940 no 3 434.9000 435.0000 nopi@cubesatsimbeta:~/CubeSat                                                                                                                                   Sim $ git status
On branch beta
Your branch is up to date with 'origin/beta'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        command_count.txt
        command_tx
        telem.txt
        telem_string.txt

nothing added to commit but untracked files present (use "git add" to track)
pi@cubesatsimbeta:~/CubeSatSim $ ./config
CubeSatSim v1.3.2 configuration tool

Mode is FSK

Simulated Telemetry is OFF

Balloon mode is OFF

Current command count is: 0

Squelch level is: 3

Radio command and control is OFF

Battery saver mode is OFF

Current sim.cfg configuration file:
W7DGP 21 47.7466 -122.1940 no 3 434.9000 435.0000 no

To change, include an OPTION
To see options, type config -h

Unknown option.  Try config -h for a list of supported options.

pi@cubesatsimbeta:~/CubeSatSim $

Ok, next is output from the ./log command. I'll try piping it to more and get it a page at a time:

pi@cubesatsimbeta:~/CubeSatSim $ ./log | more

Log file script for CubeSatSim

-- Journal begins at Sat 2024-01-27 07:45:13 PST, ends at Sat 2024-03-02 21:25:56 PST. --
Jan 27 07:51:42 cubesatsim systemd[1]: Started CubeSatSim service.
Jan 27 07:51:42 cubesatsim cubesatsim[250]: Pi test result: 902120
Jan 27 07:51:42 cubesatsim cubesatsim[250]:  39  30
Jan 27 07:51:47 cubesatsim cubesatsim[250]: Config file /home/pi/CubeSatSim/sim.cfg contains AMSAT 1 0.000000 0.000000 no 3 434.9000 435.0000 no
Jan 27 07:51:47 cubesatsim cubesatsim[250]: Transmit on 434.9000 Receive on 435.0000
Jan 27 07:51:48 cubesatsim sudo[415]:     root : PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/systemctl restart rpitx
Jan 27 07:51:48 cubesatsim sudo[415]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)
Jan 27 07:51:48 cubesatsim sudo[415]: pam_unix(sudo:session): session closed for user root
Jan 27 07:51:48 cubesatsim cubesatsim[250]: Pi Zero 2 detected
Jan 27 07:51:48 cubesatsim cubesatsim[250]: CubeSatSim v1.3.2 starting...
Jan 27 07:51:48 cubesatsim cubesatsim[250]: Test bus 1
Jan 27 07:51:49 cubesatsim cubesatsim[250]: I2C Bus Tested: /dev/i2c-1
Jan 27 07:51:49 cubesatsim cubesatsim[250]: Test bus 3
Jan 27 07:51:50 cubesatsim cubesatsim[250]: I2C Bus Tested: /dev/i2c-3
Jan 27 07:51:50 cubesatsim cubesatsim[250]: Finished testing
Jan 27 07:51:50 cubesatsim cubesatsim[250]: Battery saver mode is OFF!
Jan 27 07:51:51 cubesatsim sudo[429]:     root : PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/rm /home/pi/CubeSatSim/camera_out.jpg
Jan 27 07:51:51 cubesatsim sudo[430]:     root : PWD=/home/pi/CubeSatSim ; USER=root ; COMMAND=/usr/bin/rm /home/pi/CubeSatSim/camera_out.jpg.wav
Jan 27 07:51:51 cubesatsim sudo[429]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)
Jan 27 07:51:51 cubesatsim sudo[430]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)
Jan 27 07:51:51 cubesatsim sudo[429]: pam_unix(sudo:session): session closed for user root
Jan 27 07:51:51 cubesatsim sudo[430]: pam_unix(sudo:session): session closed for user root
Jan 27 07:52:04 cubesatsim cubesatsim[250]: pythonConfigStr: python3 -u /home/pi/CubeSatSim/ina219.py 1 3 c
Jan 27 07:52:10 cubesatsim cubesatsim[250]: pythonStr result:  0.000   0.000  0.000   0.000  4.404  -376.000  0.000   0.000  0.000   0.000  0.000   0.000  0.876  -0.300  0.000   0.000
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Mode file /home/pi/CubeSatSim/.mode contains f
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Mode is FSK
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Opened telem file
Jan 27 07:52:10 cubesatsim cubesatsim[250]: v1 Present with UHF BPF
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Tx LED Off
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Power LED On
Jan 27 07:52:10 cubesatsim cubesatsim[250]: INFO: I2C bus status 0: -1 1: 1 3: 3 camera: -1
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Serial opened to Pico
Jan 27 07:52:10 cubesatsim cubesatsim[250]:  FSK Mode, 1030 bits per frame, 200 bits per second, 5150 ms per frame, 4650 ms sample period
Jan 27 07:52:10 cubesatsim cubesatsim[250]: first time - no sleep
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Command count: 0
Jan 27 07:52:10 cubesatsim cubesatsim[250]: TX Antenna Deployed!
Jan 27 07:52:10 cubesatsim cubesatsim[250]: RX Antenna Deployed!
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Opening socket!
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Connection Failed
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Error: Connection refused
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Connection Failed
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Error: Connection refused
Jan 27 07:52:10 cubesatsim cubesatsim[250]: INFO: Reset Count: 2 Uptime since Reset: 30
Jan 27 07:52:10 cubesatsim cubesatsim[250]: ++++ Loop time: 0.003 sec +++++
Jan 27 07:52:10 cubesatsim cubesatsim[250]: Python read Result:  0.000   0.000  0.000   0.000  4.408  -330.100  0.000   0.000  0.000   0.000  0.000   0.000  0.876  -0.300  0.000   0.000
Jan 27 07:52:11 cubesatsim cubesatsim[250]: _START_FLAG_OK BME280 26.94 1009.10 34.63 26.45 MPU6050 -0.07 -0.10 -0.22 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 31.50_END_FLAG_ Payload length: 112
Jan 27 07:52:11 cubesatsim cubesatsim[250]: get_payload_status: 1
Jan 27 07:52:11 cubesatsim cubesatsim[250]: String: OK BME280 26.94 1009.10 34.63 26.45 MPU6050 -0.07 -0.10 -0.22 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 31.50
Jan 27 07:52:11 cubesatsim cubesatsim[250]: INFO: Battery voltage:  4.41 V  Threshold  3.70 V Current: -330.1 mA Threshold:  100.0 mA
Jan 27 07:52:14 cubesatsim cubesatsim[250]:  Response from STEM Payload board: OK BME280 26.94 1009.10 34.63 26.45 MPU6050 -0.07 -0.10 -0.22 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 31.50
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Writing payload string
Jan 27 07:52:14 cubesatsim cubesatsim[250]: TimeStamp: Sat Jan 27 10:52:11 2024
Jan 27 07:52:14 cubesatsim cubesatsim[250]: sensor: 0.000000 sensor: 0.000000 sensor: 26.940001 sensor: 1009.099976 sensor: 34.630001 sensor: 26.450001 sensor: 0.000000 sensor: -0.070000 sensor: -0.100000 sensor
: -0.220000 sensor: -0.090000 sensor: -0.110000 sensor: 1.040000 sensor: 0.000000 sensor: 0.000000 sensor: 0.000000 sensor: 0.000000
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Writing telem_string.txt
--More--
Jan 27 07:52:14 cubesatsim cubesatsim[250]: first time - no sleep
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Command count: 0
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Opening socket!
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Connection Failed
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Error: Connection refused
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Connection Failed
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Error: Connection refused
Jan 27 07:52:14 cubesatsim cubesatsim[250]: INFO: Reset Count: 2 Uptime since Reset: 33
Jan 27 07:52:14 cubesatsim cubesatsim[250]: ++++ Loop time: 3.753 sec +++++
Jan 27 07:52:14 cubesatsim cubesatsim[250]: Python read Result:  0.000   0.000  0.000   0.000  4.400  -332.600  0.000   0.000  0.000   0.000  0.000   0.000  0.864   0.200  0.000   0.000
Jan 27 07:52:15 cubesatsim cubesatsim[250]: _START_FLAG_OK BME280 26.94 1009.06 34.95 26.42 MPU6050 -0.03 -0.27 -0.04 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 33.00_END_FLAG_ Payload length: 112
Jan 27 07:52:15 cubesatsim cubesatsim[250]: get_payload_status: 1
Jan 27 07:52:15 cubesatsim cubesatsim[250]: String: OK BME280 26.94 1009.06 34.95 26.42 MPU6050 -0.03 -0.27 -0.04 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 33.00
Jan 27 07:52:15 cubesatsim cubesatsim[250]: INFO: Battery voltage:  4.40 V  Threshold  3.70 V Current: -332.6 mA Threshold:  100.0 mA
Jan 27 07:52:18 cubesatsim cubesatsim[250]:  Response from STEM Payload board: OK BME280 26.94 1009.06 34.95 26.42 MPU6050 -0.03 -0.27 -0.04 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 33.00
Jan 27 07:52:18 cubesatsim cubesatsim[250]: Writing payload string
Jan 27 07:52:18 cubesatsim cubesatsim[250]: TimeStamp: Sat Jan 27 10:52:15 2024
Jan 27 07:52:18 cubesatsim cubesatsim[250]: sensor: 0.000000 sensor: 0.000000 sensor: 26.940001 sensor: 1009.059998 sensor: 34.950001 sensor: 26.420000 sensor: 0.000000 sensor: -0.030000 sensor: -0.270000 sensor
: -0.040000 sensor: -0.090000 sensor: -0.110000 sensor: 1.040000 sensor: 0.000000 sensor: 0.000000 sensor: 0.000000 sensor: 0.000000
Jan 27 07:52:18 cubesatsim cubesatsim[250]: Writing telem_string.txt
Jan 27 07:52:18 cubesatsim cubesatsim[250]: first time - no sleep
Jan 27 07:52:18 cubesatsim cubesatsim[250]: Command count: 0
Jan 27 07:52:18 cubesatsim cubesatsim[250]: Opening socket!
Jan 27 07:52:18 cubesatsim cubesatsim[250]: Connection Failed
Jan 27 07:52:18 cubesatsim cubesatsim[250]: Error: Connection refused
Jan 27 07:52:18 cubesatsim cubesatsim[250]: socket send 1 1 ms bytes: 465602
Jan 27 07:52:18 cubesatsim cubesatsim[250]: socket send 2 in 1 ms bytes: 465602
Jan 27 07:52:18 cubesatsim cubesatsim[250]: socket send 3 in 0 ms bytes: 465602
Jan 27 07:52:18 cubesatsim cubesatsim[250]: socket send 4 in 1 ms bytes: 465602
Jan 27 07:52:18 cubesatsim cubesatsim[250]: socket send 5 in 1 ms bytes: 465602
Jan 27 07:52:18 cubesatsim cubesatsim[250]: INFO: Reset Count: 2 Uptime since Reset: 38
Jan 27 07:52:18 cubesatsim cubesatsim[250]: ++++ Loop time: 4.305 sec +++++
Jan 27 07:52:18 cubesatsim cubesatsim[250]: Python read Result:  0.000   0.000  0.000   0.000  4.388  -357.500  0.000   0.000  0.000   0.000  0.000   0.000  0.864   0.400  0.000   0.000
Jan 27 07:52:19 cubesatsim cubesatsim[250]: _START_FLAG_OK BME280 26.94 1009.05 34.84 26.40 MPU6050 0.32 -0.30 0.07 -0.09 -0.11 1.03 GPS 0.0000 0.0000 0.00 TMP 33.00_END_FLAG_ Payload length: 110
Jan 27 07:52:19 cubesatsim cubesatsim[250]: get_payload_status: 1
Jan 27 07:52:19 cubesatsim cubesatsim[250]: String: OK BME280 26.94 1009.05 34.84 26.40 MPU6050 0.32 -0.30 0.07 -0.09 -0.11 1.03 GPS 0.0000 0.0000 0.00 TMP 33.00
Jan 27 07:52:19 cubesatsim cubesatsim[250]: INFO: Battery voltage:  4.39 V  Threshold  3.70 V Current: -357.5 mA Threshold:  100.0 mA
Jan 27 07:52:23 cubesatsim cubesatsim[250]:  Response from STEM Payload board: OK BME280 26.94 1009.05 34.84 26.40 MPU6050 0.32 -0.30 0.07 -0.09 -0.11 1.03 GPS 0.0000 0.0000 0.00 TMP 33.00
Jan 27 07:52:23 cubesatsim cubesatsim[250]: Writing payload string
Jan 27 07:52:23 cubesatsim cubesatsim[250]: TimeStamp: Sat Jan 27 10:52:19 2024
Jan 27 07:52:23 cubesatsim cubesatsim[250]: sensor: 0.000000 sensor: 0.000000 sensor: 26.940001 sensor: 1009.049988 sensor: 34.840000 sensor: 26.400000 sensor: 0.000000 sensor: 0.320000 sensor: -0.300000 sensor:
 0.070000 sensor: -0.090000 sensor: -0.110000 sensor: 1.030000 sensor: 0.000000 sensor: 0.000000 sensor: 0.000000 sensor: 0.000000
Jan 27 07:52:23 cubesatsim cubesatsim[250]: Writing telem_string.txt
Jan 27 07:52:23 cubesatsim cubesatsim[250]: Sleep period: 3605
Jan 27 07:52:24 cubesatsim cubesatsim[250]: Command count: 0
Jan 27 07:52:24 cubesatsim cubesatsim[250]: socket send 1 1 ms bytes: 465602
Jan 27 07:52:24 cubesatsim cubesatsim[250]: INFO: Reset Count: 2 Uptime since Reset: 43
Jan 27 07:52:24 cubesatsim cubesatsim[250]: ++++ Loop time: 5.793 sec +++++
Jan 27 07:52:24 cubesatsim cubesatsim[250]: Python read Result:  0.000   0.000  0.000   0.000  4.400  -346.200  0.000   0.000  0.000   0.000  0.000   0.000  0.864   0.500  0.000   0.000
Jan 27 07:52:25 cubesatsim cubesatsim[250]: _START_FLAG_OK BME280 26.95 1009.07 34.89 26.40 MPU6050 -0.38 0.14 -0.37 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 31.50_END_FLAG_ Payload length: 111
Jan 27 07:52:25 cubesatsim cubesatsim[250]: get_payload_status: 1
Jan 27 07:52:25 cubesatsim cubesatsim[250]: String: OK BME280 26.95 1009.07 34.89 26.40 MPU6050 -0.38 0.14 -0.37 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 31.50
Jan 27 07:52:25 cubesatsim cubesatsim[250]: INFO: Battery voltage:  4.40 V  Threshold  3.70 V Current: -346.2 mA Threshold:  100.0 mA
Jan 27 07:52:28 cubesatsim cubesatsim[250]:  Response from STEM Payload board: OK BME280 26.95 1009.07 34.89 26.40 MPU6050 -0.38 0.14 -0.37 -0.09 -0.11 1.04 GPS 0.0000 0.0000 0.00 TMP 31.50
Jan 27 07:52:28 cubesatsim cubesatsim[250]: Writing payload string
Jan 27 07:52:28 cubesatsim cubesatsim[250]: TimeStamp: Sat Jan 27 10:52:25 2024
Jan 27 07:52:28 cubesatsim cubesatsim[250]: sensor: 0.000000 sensor: 0.000000 sensor: 26.950001 sensor: 1009.070007 sensor: 34.889999 sensor: 26.400000 sensor: 0.000000 sensor: -0.380000 sensor: 0.140000 sensor:
 -0.370000 sensor: -0.090000 sensor: -0.110000 sensor: 1.040000 sensor: 0.000000 sensor: 0.000000 sensor: 0.000000 sensor: 0.000000
Jan 27 07:52:28 cubesatsim cubesatsim[250]: Writing telem_string.txt

Ok, that's all of it.
Thanks and 73,
Larry W7DGP

**Reply by W7DGP** [reply]
Alan, I notice when I saved the file a line has been drawn through the cat command and output for cat sim.cfg. That did not appear on the screen. That is the actual output. I have no idea why that line is there.
-Larry

**Comment by alanbjohnston** [comment]
I found the bug in the python code and fixed it.

If you are running the latest beta software, this will be fixed.  You can make sure by typing this:

```
cd
cd CubeSatSim
git pull
git checkout beta
make debug
sudo reboot now
```

Thanks for finding this bug!

73,
Alan
KU2Y
